### PR TITLE
Invisible Google reCaptcha

### PR DESCRIPTION
**Pull request by thumperca**
Commit: https://github.com/vibhu-athavaria/gogovgo/commit/b13bf587c01ef02473db92e93ad61bd960a3943a

Edit: Also added https://github.com/thumperca/gogovgo/commit/a7eea224d0ce171a0e4f283ada33387f3362f869

Edit2: 
Added https://github.com/vibhu-athavaria/gogovgo/pull/10/commits/37bde01e25098ecb0cb939b0b20872d5745b75b2 and https://github.com/vibhu-athavaria/gogovgo/pull/10/commits/40fccb6dd01128c32e515cdb016b537f92e9090f
See https://trello.com/c/cFXgIEtq/8-add-google-maps-api-address-validation-for-new-reviews for details

### CODE REVIEW

**Comment by vibhu-athavaria** [comment]
Please create a new branch after updating your local with origin/master. Apply your current changes.

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,35 +1,53 @@
...
+import LandingPage from "./Application/LandingPage";
 
 class App extends Component {
+    render() {
+        /**
+         * Component to redirect traffic from homepage to /politician/us/president-united-states
+         * @param {object} props
+         */
+        const HomeRedirect = props => {
+            return <Redirect to="/politician/us/president-united-states" />;
```
[the remark this answers is not in the document]
Comment (thumperca) [reply frontend/src/App.js]: Nowhere, this is to redirect any traffic from / to President's page. HomeRedirect is a dump-component.

[the remark this answers is not in the document]
Comment (thumperca) [reply frontend/src/App.js]: I've pushed another commit as well. Will use branches for any future commit.

**Comment by vibhu-athavaria** [comment]
Ok I am merging this PR. Please create a new branch from now on.

**Pull request merged**